config/environments/development.rb

```diff
@@ -55,3 +55,21 @@
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 end
+
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider: 'AWS', # required
+    aws_access_key_id: ENV['AWS_ACCESS_KEY_ID'], # required unless using use_iam_profile
+    aws_secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'], # required unless using use_iam_profile
```

Comment:
We can very handily set these in `config/secrets.yml` for each environment. You can then access them in your code like this:

```suggestion
    aws_access_key_id: Rails.application.secrets.aws[:key_id], # required unless using use_iam_profile
    aws_secret_access_key: Rails.application.secrets.aws[:access_key], # required unless using use_iam_profile
```

I can show you how to add these to the production machine as well before deploying

Reply:
Updated the logic here. For production, we don't need to use secrets since we can use AWS to do it for us using IAM roles. I'll explain when we sync later today.